mautic/mautic
# Remove points from properties for contacts import

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | no
| Deprecations?                          | no
| BC breaks? (use the c.x branch)        | no
| Automated tests included?              | no
| Related user documentation PR URL      | no need
| Related developer documentation PR URL | no need
| Issue(s) addressed                     | 



#### Description:

Contact's import has points option twice. This PR remove it from properties 

![image](https://user-images.githubusercontent.com/462477/142844516-5bf7e93d-0e63-44f9-9634-0388fcd6bf79.png)




#### Steps to test this PR:


1. Load up [this PR](https://mautibox.com)
2. Create import for contacts
3. Try map points
4. Before PR you see points two times, after PR just once
5. Check if works like before




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10635"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
